flytekit/extend/__init__.py

```diff
@@ -0,0 +1,9 @@
+from flytekit.common.translator import get_serializable
```

Comment:
Nice, will we document this?

Reply:
not now.  but the goal is that anything that's in any of these three directories will have documentation as extensive as the `FlyteFile` one.